h2o-py/tests/testdir_algos/glm/pyunit_PUBDEV_7481_glm_startvals.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator as glm
+
+# test startval to set GLM coefficients
+def set_glm_startvals():
+    # read in the dataset and construct training set (and validation set)
+    d = h2o.import_file(path=pyunit_utils.locate("smalldata/logreg/prostate.csv"))
+    mL = glm(family='binomial')
+    mL.train(training_frame=d,x=[2,3,4,5,6,7,8],y=1)
+    mLcoeff = mL.coef()
+    r = glm.getGLMRegularizationPath(mL)
+    rcoeff = r["coefficients"][0]
+    startval1 = [0,0,0,0,0,0,0,-0.3945120960889672]
+    startval2 = [rcoeff["AGE"], rcoeff["RACE"], rcoeff["DPROS"], rcoeff["DCAPS"], rcoeff["PSA"], rcoeff["VOL"], 
+                rcoeff["GLEASON"], rcoeff["Intercept"]]
+    startvalBad = [0,0]
+    
+    ml1 = glm(family="binomial", startval = startval1) # same starting condition as GLM
+    ml1.train(training_frame=d,x=[2,3,4,5,6,7,8],y=1)
+    ml1Coeff = ml1.coef()
+    pyunit_utils.assertEqualCoeffDicts(mLcoeff, ml1Coeff , tol = 1e-6) # coeffs should be the same
+
+    ml2 = glm(family="binomial", startval = startval2) # different starting condition from GLM
+    ml2.train(training_frame=d,x=[2,3,4,5,6,7,8],y=1)
+    ml2Coeff = ml2.coef()   
+    
+    try:
+        pyunit_utils.assertEqualCoeffDicts(mLcoeff, ml2Coeff , tol = 1e-6)
+        sys.exit("Should have thrown an error as coefficients are different!")        
```

Comment:
no `sys.exit` please, there may be more tests to run...
this is the pattern:
```
try:
    # statement expected to raise
    assert False, "Should have thrown an error as coefficients are different!"
except Exception as expected:
    # verify that the exception that has been thrown is the one you expect
    assert "expected message" in str(expected)
```

Reply:
I will use assert False then.

Reply:
Done